porcupine/tabs.py

```diff
@@ -709,12 +733,7 @@ def reload(self, *, undoable: bool = True) -> bool:
                         continue
 
             except UnicodeDecodeError:
-                bad_encoding = self.settings.get("encoding", str)
-                user_selected_encoding = utils.ask_encoding(
-                    f'The content of "{self.path}" is not valid {bad_encoding}. Choose an encoding'
-                    " to use instead:",
-                    bad_encoding,
-                )
+                user_selected_encoding = confirm_encoding("latin1")
```

Comment:
Why does this hard-code latin1, an encoding used mostly in just one part of the world?